### PR TITLE
vector reserve before push_back

### PR DESCRIPTION
**Pull request by sduzh**

### CODE REVIEW

File: be/src/olap/rowset/segment_v2/segment_writer.cpp [hunk 1]
```diff
@@ -49,7 +49,9 @@ SegmentWriter::SegmentWriter(fs::WritableBlock* wblock,
 SegmentWriter::~SegmentWriter() = default;
 
 Status SegmentWriter::init(uint32_t write_mbytes_per_sec) {
+    (void)write_mbytes_per_sec;
```
Comment (imay) [inline be/src/olap/rowset/segment_v2/segment_writer.cpp:52]:
```suggestion
Status SegmentWriter::init(uint32_t write_mbytes_per_sec __attribute__((unused))) {
```
also works

**Comment by imay** [comment]
@sduzh 
It will be better to describe what has been done in this CL in commit message.

File: be/src/olap/rowset/segment_v2/segment_writer.cpp (hunk 1 above)
Comment (sduzh) [reply be/src/olap/rowset/segment_v2/segment_writer.cpp]: updated

**Review by imay — DISMISSED** [review]
+1 LGTM

**Comment by imay** [comment]
Hi @sduzh 
Please don't push code to unrelated PR next time, better to create another PR.

**Review by kangkaisen — APPROVED** [review]
+1, LGTM

**Review by morningman — APPROVED** [review]
LGTM
P0 failed due to another bug, not related to this PR.

**Pull request merged**